include/oneapi/dpl/internal/async_impl/glue_async_impl.h

```diff
@@ -79,10 +79,12 @@ copy_async(_ExecutionPolicy&& __exec, _ForwardIterator1 __first, _ForwardIterato
 {
     auto __dispatch_tag = oneapi::dpl::__internal::__select_backend(__exec, __first, __result);
 
+    using _DecayedExecutionPolicy = std::decay_t<_ExecutionPolicy>;
+
     wait_for_all(::std::forward<_Events>(__dependencies)...);
     auto ret_val = oneapi::dpl::__internal::__pattern_walk2_brick_async(
         __dispatch_tag, ::std::forward<_ExecutionPolicy>(__exec), __first, __last, __result,
-        oneapi::dpl::__internal::__brick_copy<decltype(__dispatch_tag), _ExecutionPolicy>{});
+        oneapi::dpl::__internal::__brick_copy<decltype(__dispatch_tag), _DecayedExecutionPolicy>{});
```

Comment:
In this and many other functions, `_DecayedExecutionPolicy` is only used once in its definition scope. Meanwhile, it saves just 7 symbols, comparing to `std::decay_t<_ExecutionPolicy>`, and adds nothing to readability.

I see little value in the alias even where it is reused, and absolutely zero (that is, -273 :)) reasons to add it when used only once.

Reply:
I am ready to any variant, it's not problem....

Reply:
Fixed